andela/LIT-Kigali1
# #164104126 setup ui sass

#### What does this PR do?
 Configure SASS to be used for template designing
#### Description of Task to be completed?
- add UI folder and its necessary subfolders
- add some boilerplate CSS codes( grid system, helper classes, font, colors,...)
- add SASS preprocessor
#### How should this be manually tested?
- git clone
- npm install
- npm run sass for compiling the SASS to CSS
- open this file ./ui/index.html to see the effect
#### What are the relevant pivotal tracker stories?
#164104126
